fix(iam): new invit MTA-3660 (#2132)
scaleway/docs-content: identity-and-access-management/iam/how-to/accept-invitation-to-orga.mdx

When someone invites you to join their Organization, you receive an email to inform you.

## If you already have a Scaleway account

1. Click **Log in to your account** from your email invitation. The Scaleway console opens in a browser window.

> 3. Log into the Scaleway console using your credentials. Your Organization overview page displays.

Reviewer: Steps out of order

> 3. Log into the Scaleway console using your credentials. Your Organization overview page displays.

Reviewer 2: ```suggestion
2. Log into the Scaleway console using your credentials. Your Organization overview page displays.
```

    <Message type="note">
      If you already logged in, skip to the next step.
    </Message>

> 2. Click **Accept** in the top right pop-up.

Reviewer 2: ```suggestion
3. Click **Accept** in the top right pop-up.
```

    <Lightbox src="scaleway-accept-orga-invite.webp" alt="" />

> 

Reviewer 3: Let's discuss the AB testing part

    <Message type="important">

>       If you click **Decline**, your invitation will be rescinded. If you want to be invited again, you must ask the [IAMManager](/identity-and-access-management/iam/reference-content/permission-sets) or [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization.

Reviewer 4: ```suggestion
      If you click **Decline**, your invitation will be rescinded. If you want to be invited again, you must ask the [IAM Manager](/identity-and-access-management/iam/reference-content/permission-sets) or [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization.
```

    </Message>

    You are redirected to the overview page of the Organization to which you were invited.

    As an [IAM user](/identity-and-access-management/iam/concepts/#user) of an Organization, you must always ensure that you are in the desired Organization by checking the Organization name that displays in the top right corner of your console.

>     In the image below, we are currently in the Marie Curie Organization (highlighted in red). Use the drop-down menu to navigate to a different Organization, for example below we could switch to the Pierre Curie Organization.

Reviewer 2: Didn't we change it to Marion Créhange (instead of Marie Curie)?

    <Lightbox src="scaleway-orga-dropdown.webp" alt="" />

## If you do not have a Scaleway account

> 1. Click **Log in to your account** from your email invitation. The Scaleway console opens in a browser window.

Author: ```suggestion
1. Click **Create your account** from your email invitation. The Scaleway console opens in a browser window.
```

> 3. Create a Scaleway account. Follow the [How to create a Scaleway account](/console/my-account/how-to/create-an-account) documentation page to do so. Your Organization overview page displays.

Reviewer: Steps again

> 3. Create a Scaleway account. Follow the [How to create a Scaleway account](/console/my-account/how-to/create-an-account) documentation page to do so. Your Organization overview page displays.

Reviewer 2: ```suggestion
2. Create a Scaleway account. Follow the [How to create a Scaleway account](/console/my-account/how-to/create-an-account) documentation page to do so. Your Organization overview page displays.
```

> 2. Click **Accept** in the top right pop-up.

Reviewer 2: ```suggestion
3. Click **Accept** in the top right pop-up.
```

    <Lightbox src="scaleway-accept-orga-invite.webp" alt="" />

    <Message type="important">

>       If you click **Decline**, your invitation will be rescinded. If you want to be invited again, you must ask the [IAMManager](/identity-and-access-management/iam/reference-content/permission-sets) or [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization.

Reviewer 4: ```suggestion
      If you click **Decline**, your invitation will be rescinded. If you want to be invited again, you must ask the [IAM Manager](/identity-and-access-management/iam/reference-content/permission-sets) or [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization.
```

    </Message>

    You are redirected to the overview page of the Organization to which you were invited.

    As an [IAM user](/identity-and-access-management/iam/concepts/#user) of an Organization, you must always ensure that you are in the desired Organization by checking the Organization name that displays in the top right corner of your console.

>     In the image below, we are currently in the Marion Créange Organization (highlighted in red). Use the drop-down menu to navigate to a different Organization, for example below we could switch to the Documentation Team Organization.

Reviewer 5: Use the drop-down menu to navigate to a different Organization. For example, below, we could switch to the Documentation Team Organization.

Author: ```suggestion
    In the image below, we are currently in the Marion Créange Organization (highlighted in red). Use the drop-down menu to navigate to a different Organization. For example, below, we could switch to the Documentation Team Organization.
```
